scripts/live_test/generate_index.py

```diff
@@ -177,7 +177,7 @@ def render(data, container, container_url, testdata, USER_REPO, USER_BRANCH, COM
                 # elif 'sequential' in name:
                 #     display_name = 'sequential'
                 try:
-                    html = requests.get(url).content.__str__()
+                    html = str(requests.get(url).content)
```

Comment:
```suggestion
                    html = requests.get(url).text
```
Although `str(.content)` is not identical to `.text`. I suppose it should use `.text`.

@wangzelin007 for confirmation.

Reply:
This is very old code. I think you're right; we should use .text for the HTML content.